arangodb-net-standard/DocumentApi/PostDocumentsDocumentResponse.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System.Net;
+﻿﻿using System.Net;
```

Comment:
Do you know what has changed here? I can't see anything but maybe there's a BOM character or something else that's changed?

Reply:
Thats odd, I don't think I purposely changed anything in the Documents Client nor the Transaction. My compare highlights the word using but there is no difference. So maybe just do a git checkout of the file and undo the changes

Reply:
Can you check for a BOM (byte order mark)?  This will be the first (non-printing) character in the file.  I'd rather not commit these two changes if its something like that.